yarn.lock

```diff
@@ -9843,6 +9843,13 @@ iconv-lite@0.4.24, iconv-lite@^0.4.17, iconv-lite@^0.4.24:
   dependencies:
     safer-buffer ">= 2.1.2 < 3"
 
+idna-uts46-hx@3.4.0:
```

Comment:
is this supposed to be committed since it's an auto generated file?

Reply:
i believe yarn.lock is supposed to be committed, yeah